NutriApp/Views/Evaluation/EstimatedHeight/EstimatedHeightPageViewModel.cs

```diff
@@ -42,13 +22,31 @@ public bool CheckedMan
     [ObservableProperty]
     private bool _hasErrorKneeHeight;
 
-    public ICommand WomanCommand { get; set; }
-    public ICommand ManCommand { get; set; }
+    [RelayCommand]
+    private void HandleCheckedChange(object checkGener)
+    {
+        if ((Genders)checkGener == Genders.Male)
```

Comment:
```suggestion
        if ((Genders)checkGener is Genders.Male)
```

não dá pra fazer algo assim?